common/types/loadbalancer.go

```diff
@@ -101,15 +107,20 @@ type LoadBalancer struct {
 func (lb *LoadBalancer) AddService(svc LBSVC) bool {
 	oldSvc, ok := lb.SVCMapID[svc.FE.ID]
 	if ok {
-		// If service already existed, remove old entry from map
+		// If service already existed, remove old entry from Cilium's map
+		log.Debugf("service %s is already in lb.SVCMapID; deleting old entry and updating it with new entry", svc.FE.String())
 		delete(lb.SVCMap, oldSvc.Sha256)
 	}
+	log.Debugf("adding service %s with SHA %s to lb.SVCMap", svc.FE.String(), svc.Sha256)
```

Comment:
Do we still need all this debug messages?

Reply:
I found that the debug messages I added were useful in determining what was going on in this code. There were very few logs when I started out. I don't see a downside to having this log - it provides important information about the state managed in Cilium.